templater.sh

```diff
@@ -125,6 +125,8 @@ if [[ "$print_only" == "true" ]]; then
     exit 0
 fi
 
+# allow us to pass strings with '*' character without globbing
```

Comment:
This comment is a little unclear. Maybe include an example of what you're describing?